e2e/settings.spec.ts

```diff
@@ -300,4 +300,32 @@ test.describe('Settings', () => {
       })
     ).toBeVisible();
   });
+
+  test('Should allow empty string in any field in about settings', async ({
```

Comment:
Given that initial bug was that you couldn't change from filled in fields to empty ones (for picture at least). Does this start with some data in any of the fields? If not, could you add something and then remove it?

Since we actually check the image link, it may need mocking. If this turns out to be a massive headache, we can rethink it!

Reply:
Hm, it would have at least the name. But I've added to test changing to concrete values first (the same example data in the api test).

Reply:
Yeah, that works, thanks. If we can worry about the flakiness if Quincy's signature randomly starts to disappear.